### PR TITLE
Modify the requirements.txt and the README file to adding the honcho …

### PR DESCRIPTION
**Pull request by GaryAvc**
modify requirements.txt and readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,38 +102,42 @@ Example Request:
...
+	"meta": "{\"dollarsOff\":10}",
+	"begin_date": "18-Nov-2018 (08:34:58.674035)",
+	"end_date": null,
+	"active": true
 }
 ```
 
+```shell
+    $ honcho start
+
```
Comment (dizys) [inline README.md:135]: 微信截图_20211115171838
This is causing disorder. Please fix this and also meaningfully title this section as well.

File: README.md [hunk 2]
```diff
@@ -23,18 +23,18 @@ Example response:
...
-    "name": "Amazing",
-    "type": "coupon",
-    "description": "Amazing $10 coupon for toasters",
-    "meta": "{\"dollarsOff\":10}",
-    "begin_date": "18-Nov-2018 (08:34:58.674035)",
-    "end_date": null,
-    "active": true
-  }
-  // ...
+	{
```
Comment (dizys) [inline README.md:26]: I don't see the point of adding more spaces as indent here. Do you mind changing them back?
Same as below.

**Review by dizys — CHANGES_REQUESTED** [review]

**Comment by GaryAvc** [comment]
Modify the README file to make sure everything shows properly :)

File: README.md [hunk 3]
```diff
@@ -102,38 +102,44 @@ Example Request:
...
+	"end_date": null,
+	"active": true
 }
 ```
 
+honcho instructions:
+
+```shell
+     honcho start
+
```
Comment (dizys) [inline README.md:137]: Unfortunately, this is missing ```. So its still messing up with rest of the README file.
微信截图_20211115171838

**Review by dizys — CHANGES_REQUESTED** [review]

**Review by dizys — APPROVED** [review]

**Comment by dizys** [comment]
I fixed the problem myself. Please start earlier and leave more time for code review and changes in later sprints.

**Pull request merged**